### DISCUSSION TITLE
Documentation correction needed

### DISCUSSION THREAD

**Discussion by jake-luna-sada** [mondoohq/cnquery#2656] [General]
Playing around with with cnspec for Terraform and noticed some issues in the docs that need updating 

<img width="1287" alt="Screenshot 2023-11-27 at 9 08 57 AM" src="https://github.com/mondoohq/cnquery/assets/91563569/12e52e87-b766-402c-bcc7-6f9d84aca3f4">
<img width="1194" alt="Screenshot 2023-11-27 at 9 09 50 AM" src="https://github.com/mondoohq/cnquery/assets/91563569/8a5f4fc9-3b14-4e36-9cdd-dc5a536ea994">
<img width="1034" alt="Screenshot 2023-11-27 at 9 09 03 AM" src="https://github.com/mondoohq/cnquery/assets/91563569/8d0af829-07f6-432e-aa30-d9035ff74908">

https://mondoo.com/docs/cnspec/supplychain/terraform/ <--where to update

**Comment by imilchev** [comment]
You are right! The docs seem to be outdated. This is now just `cnspec scan terraform /path/to/terraform/` 

cc @misterpantz

**Reply by chris-rock** [reply]
Created an issue and assigned it to @misterpantz https://github.com/mondoohq/cnquery/issues/2658

**Comment by misterpantz** [comment]
Thank you, @Jluna-sada. I released the fix this morning.
